lib/markdown-renderer.ts

```diff
@@ -36,6 +36,10 @@ export function md2text(markdown: string, columns = 76): string {
                         .replace(/(^|\n)(\n)(?!$)/g, "$1>$2"); // quote empty
                     }});
             },
+            checkBoxFormatter: (elem, _walk, builder) => {
+                const attribs = elem.attribs as { checked?: boolean };
```

Comment:
The cast looks tricky -- hopefully just to me, lacking TypeScript experience. It would be nice if the type of elem could be inferred from the caller of this callback, but that's probably a bridge too far. elem.attrib also has a property named "type" (not optional, it seems), but we can simply ignore it here, I guess? https://github.com/html-to-text/node-html-to-text/blob/ba8697718d9995efbfe6255fe27d629e9d556b0b/lib/formatter.js#L147 has "const attribs = elem.attribs || {};"; would that work as well?

Reply:
This cast basically says that we know that this `any` type is actually an object that has an optional `checked` attribute.

However, there is a bug: the attribute has type string ;-)

Will fix.